andela/odin-ah-frontend
# #160452440 Users should be able to comment on an article

#### What does this PR do?
- Implements ability for user to post comment and see comments on an article
#### Description of Task to be completed?
- Logged in user should be able to post a comment on an article
- Any user should be able to see comments on an article
#### How should this be manually tested?
- Start the application using npm start
- Go to to [localhost:3000](http://localhost:3000)
- Login with an existing account
- On the list of articles section, click on any article of your choice
- `http://localhost:3000/article/{article-slug}`
- Scroll down to see the comment input filed, post a comment
- See the comment in the comment list section of the page
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[#160452440](https://www.pivotaltracker.com/story/show/160452440)
#### Screenshots (if appropriate)
![screen shot 2018-11-22 at 8 06 10 pm](https://user-images.githubusercontent.com/9039613/48919318-6449e380-ee92-11e8-8f7a-543b6eabca57.png)
![screen shot 2018-11-22 at 8 07 03 pm](https://user-images.githubusercontent.com/9039613/48919325-6c098800-ee92-11e8-9dd0-4baa20f1777b.png)
#### Questions:
